docs/docs/README.md

```diff
@@ -1,55 +1,50 @@
 # Infrastructure for Atlassian Data Center products on Kubernetes
+
 [![Atlassian license](https://img.shields.io/badge/license-Apache%202.0-blue.svg?style=flat-square)](https://github.com/atlassian-labs/data-center-terraform/blob/main/LICENSE) 
 [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](https://github.com/atlassian-labs/data-center-terraform/blob/main/CONTRIBUTING.md)
 
-!!! warning "This project is still under development and is not officially supported."
+## Support disclaimer
+
+!!! warning "Supported Products and Platforms"
+    **This project is still under development and is not officially supported.**
 
-[Atlassian DC Apps program](https://developer.atlassian.com/platform/marketplace/dc-apps-submitting-your-app/#step-2--test-your-app-s-performance-at-scale)
-provides App vendors in Atlassian ecosystem with tools to setup ready-to-use environment. 
-This project provides a tool to provision infrastructure for Atlassian DC Helm chart products.
-At this stage the scope is providing the infrastructure for Bamboo DC.
+    Current project limitations listed below:
 
+    * [AWS](https://aws.amazon.com/){.external} is the only supported cloud provider.
+    * [Bamboo DC](https://confluence.atlassian.com/bamboo/bamboo-8-1-release-notes-1103070461.html){.external} is the only DC product supported by this project.
 
-## Prerequisites
+    Support for additional Cloud providers and DC products will be made available in future.
+    
+This project can be used for bootstrapping Atlassian Bamboo DC in a K8s cluster. Soup to nut's, this tool will stand-up a Kubernetes cluster and all the required infrastructure needed for running Bamboo DC. Using the existing [Data Center Helm Charts](https://atlassian.github.io/data-center-helm-charts/#additional-content){.external} it will also install Bamboo DC into this pre-provisioned cluster.
 
-In order to deploy the infrastructure for Atlassian Data Center products on Kubernetes you need to have the 
-following applications installed on your local machine:
+## Deployment overview
 
-* AWS CLI
-* Helm
-* Terraform
+The diagram below provides a high level overview of what a typical deployment will look like:
 
-See [prerequisites](userguide/PREREQUISITES.md) for details. 
+![architecture](assets/images/Terraform.png "Deployment")
```

Comment:
`S3` and `DynamoDB` are not provisioned in the vpc. So technically although they are defined peivate and encrypted, we shouldn't have them in vpc.

Reply:
Will fix